digital_asset_types/src/dapi/common/asset.rs

```diff
@@ -63,6 +64,7 @@ pub fn build_asset_response(
     };
     let (items, errors) = asset_list_to_rpc(assets, transform);
     AssetList {
+        // grand_total: grand_total.unwrap(), // TODO: Add this back in
```

Comment:
I'd add clarify that we are not adding this for now because we don't want to commit to the schema.